### ISSUE TITLE
Strange mean and std for inception_v4.

### ISSUE THREAD

**Issue by KimalIsaev** [creafz/pytorch-cnn-finetune#20]
Hi, and sorry if it's stupid question.
But value for mean of inception_v4 equals [0.5, 0.5, 05].
And value for std of inception_v4 equals [0.5, 0.5, 05].
Transformation of tensor by Normalize with this values map 0 to -1 and 255 to 509.
Is it normal?

**Comment by KimalIsaev** [comment]
Here test file.
[model_info_test.zip](https://github.com/creafz/pytorch-cnn-finetune/files/4451426/model_info_test.zip)
It just prints mean and std of inception_v4

**Comment by KimalIsaev** [comment]
It's stupid question, answer can be found here: https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html
